documentation/src/main/antora/modules/ROOT/pages/tutorial-create-my-first-component.adoc

```diff
@@ -0,0 +1,165 @@
+= Creating your first component
+:page-partial:
+
+This tutorial walks you through all the required steps to get started with *Talend Component Kit*, from the creation of a simple component to its integration into *Talend Studio*.
+
+The component created in this tutorial is a simple output component that only reads the data it receives and displays it in the logs, along with an extra information entered by a user.
```

Comment:
i will not say that the component reads the data.
it more : **receive  data from the previous component**